gamestonk_terminal/README.md

```diff
@@ -427,6 +427,7 @@ Command|Description|Source
 `load`          | Load new ticker
 `info`          | Display option information | [Barchart](https://barchart.com/)
 `calc`          | Basic option PnL calculator
+`act`           | Scrapes unusual options activity|
```

Comment:
Can we call it "unu" and change desc to "Unusual options activity".

My reasoning is that act would imply most active options. But since they call it unusual, I'm assuming they compare the activity with the past and if it's much different then it's considered unusual.

E.g. if Apple is traded a lot yesterday, and today no trades. That's unusual, and "activity" wouldn't reflect this so well. But this is pure speculation :)

Reply:
I can. I think this source refers to unusual as just high Vol/OI (which is a typical description)